### ISSUE TITLE
Font shorthand property line-height bug

### ISSUE THREAD

**Issue by medialwerk** [jakubpawlowicz/clean-css#548]
I guess i have found a bug at the font shorthand property. At the following sample the line-height compiles to the font-weight:
~~~
.my-class {
  font: 11px/normal sans-serif;
}
~~~
compiles to:
~~~
.my-class{font:11px/400 sans-serif}
~~~
If i would add a font-weight it is working:
~~~
.my-class {
  font: 400 11px/normal sans-serif;
}
~~~
compiles to:
~~~
.my-class{font:400 11px/normal sans-serif}
~~~
Thanks!

**Comment by jakubpawlowicz** [comment]
Great catch of another edge case!

**Comment by jakubpawlowicz** [comment]
Fix is on 3.2 branch. I'll try to fix one more ticket and it's out.

**Comment by jakubpawlowicz** [comment]
It's out in 3.2.5.
